identity_cache.gemspec

```diff
@@ -31,6 +31,6 @@ Gem::Specification.new do |gem|
     gem.add_development_dependency('cityhash', '0.6.0')
     gem.add_development_dependency('mysql2')
     gem.add_development_dependency('pg')
-    gem.add_development_dependency('stackprof') if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.1.0")
+    gem.add_development_dependency('stackprof') if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new("2.1.0")
```

Comment:
I am running 1.9.3 on production. While the plan is to get to 2+ as soon as possible, it's not as trivial as it would seem. At least in my team's case.

My feeling is that `identity_cache` is quite a popular project and I wouldn't be surprised to learn that a lot of people still using it are running on 1.9, especially since it's a version still supported by the latest version of Rails.

I don't mean to tell you how to run your project of course, but since identity_cache is very tied to Rails, maybe it would make sense to have the ruby dependencies kind of follow Rails'. I see `Gemfile`s for versions going back to 3.2, which I don't believe supports ruby 2.

That being said, I am also all for moving forward, a pattern I've seen in some OSS projects is to lock a version for some older versions of ruby or rails and master has its own future forward dependencies.

My long way of saying that yes, I use 1.9, and I hope you'll still support it for a little longer :)